Update the page on "FHIR Connector Example" (#423)
wso2/docs-mi: en/docs/reference/connectors/fhir-connector/fhir-connector-example.md

> 1. `/create` : creates a new patient at FHIR server

Reviewer: ```suggestion
1. `/create` : create a new patient at FHIR server
```

2. `/read` : retrieve information about the patient from FHIR server
3. `/readSpecificResourceById`: read patient by Id
4. `/update` : update patient information from FHIR server.
5. `/delete` : remove added patient information from FHIR server.

To know further information about these operations please refer this link.

> > **Note**: If no ID element is provided, or the value is wrong, the server responds with a HTTP 400 error code and provides an operation outcome identifying the issue.

Reviewer: ```suggestion
> **Note**: If the ID element is not provided, or the value is wrong, the server responds with a HTTP 400 error code and provides an operation outcome identifying the issue.
```

The following diagram illustrates all the required functionality of the FHIR API service that you are going to build.

> In here FHIR clients can invoke the API with HTTP/HTTPs with the required information. The FHIR connector exposes each request by converting to the Health Level Seven International standards and then send to the resources available in the FHIR server.

Reviewer: ```suggestion
In here FHIR clients can invoke the API with HTTP/HTTPs with the required information. The FHIR connector processes each request by converting to the Health Level Seven International standards and then sends it to the resources available on the FHIR server.
```

               <log level="full" separator=","/>
               <send/>
           </inSequence>
           <faultSequence/>
       </resource>
   </api>
   ```
    To learn about supported operations and their parameters, please refer to `FHIR connector reference`.

Now we can export the imported connector and the API into a single CAR application. The CAR application is what we are going to deploy during server runtime.

## Export integration logic as a CApp
In order to export the project, refer to the [build and export the carbon application]({{base_path}}/develop/deploy-artifacts/#build-and-export-the-carbon-application) guide. 

## Get the project

> 

Reviewer: Modify the zip file

Follow these steps to deploy the exported CApp in the integration runtime. 

> **Deployment on Micro Integrator**

Reviewer: ```suggestion
**Deploy on Micro Integrator**
```

In order to deploy and run the project, refer the [build and run]({{base_path}}/develop/deploy-artifacts/#build-and-run) guide.

> You can further refer the application deployed through the CLI tool. See the instructions on [managing integrations from the CLI]({{base_path}}/observe-and-manage/managing-integrations-with-apictl).

Reviewer: This path is changed. Can you check and update
